widgets/notmuch_all.lua

```diff
@@ -0,0 +1,42 @@
+-- notmuch_all - count messages that match a notmuch query
+-- Copyright (C) 2019  Enric Morales
+--
+-- This file is part of Vicious.
+--
+-- Vicious is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as
+-- published by the Free Software Foundation, either version 2 of the
+-- License, or (at your option) any later version.
+--
+-- Vicious is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with Vicious.  If not, see <https://www.gnu.org/licenses/>.
+
+local tonumber = tonumber
+local helpers = require "vicious.helpers"
+local spawn = require "vicious.spawn"
+
+
+local notmuch = {}
+local output = {}
+
+local function parse(stdout, stderr, exitreason, exitcode)
+   if exitcode == 0 then
+      output.count = tonumber(stdout)
+   else
+      output.count = "N/A"
+   end
+   return output
```

Comment:
Since `output` is solely processed within this function, please declare it within the scope, i.e.

```lua
local output = { count = "N/A" }
if exitcode == 0 then output.count = tonumber(stdout) end
return output
```

Reply:
Done! Thanks!